src/ddscxx/include/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.hpp

```diff
@@ -264,6 +266,18 @@ DataWriterQosDelegate::policy<dds::core::policy::WriterDataLifecycle>()
     return lifecycle_;
 }
 
+
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.